calculate_average_royvanrijn.sh

```diff
@@ -16,5 +16,11 @@
 #
 
 
-JAVA_OPTS=""
-time java $JAVA_OPTS --class-path target/average-1.0.0-SNAPSHOT.jar dev.morling.onebrc.CalculateAverage_royvanrijn
+# Added for fun, doesn't seem to be making a difference...
+if [ -f "target/calculate_average_royvanrijn.jsa" ]; then
+    JAVA_OPTS="-XX:SharedArchiveFile=target/calculate_average_royvanrijn.jsa -Xshare:on"
+else
+    # First run, create the archive:
+    JAVA_OPTS="-XX:ArchiveClassesAtExit=target/calculate_average_royvanrijn.jsa"
+fi
+java $JAVA_OPTS --class-path target/average-1.0.0-SNAPSHOT.jar dev.morling.onebrc.CalculateAverage_royvanrijn
```

Comment:
Could you bring back the time call here? I figured it saves a few ms to have it here vs. in the evaluate loop script, so let's do it here in the actual scripts for the sake of consistency and comparability.

Reply:
I've re-added the `time` there and squashed everything after a long and hard battle.

Also improved the code a little with a tiny custom HashTable, because, why not? Speeeed.